doc/tools/make_rst.py

```diff
@@ -59,6 +59,7 @@
     "value",
     "Getter",
     "This method should typically be overridden by the user to have any effect.",
+    "This method is required to be overridden.",
```

Comment:
```suggestion
    "This method is required to be overridden when extending its base class.",
```

Reply:
With the extra words, the order feels a little awkward to me. Maybe:

```suggestion
    "Overriding this method is required when extending its base class.",
```

Reply:
It's awkward, true, but it was done for consistency with the other qualifiers tooltips that also begin with "This method [...]". I have no opinion either way.